### PR TITLE
#163651958 Setup Babel

### PR DESCRIPTION
**Pull request by eokwukwe**
#### What does this PR do?
Setup babel for ES6+ to ES5 transpiler
#### Description of Task to be completed?
- [x] install babel
- [x] configure the project to use babel as ES6+ to ES5 transpiler
#### What are the relevant pivotal tracker stories?
[#163651958](https://www.pivotaltracker.com/story/show/163651958)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,11 +5,18 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 0",
-    "start": "node index.js"
+    "start": "npm run build && node ./build/index.js",
+    "dev": "babel-watch index.js",
+    "build": "npm run clean && mkdir build && babel ./ --ignore 'node_modules,build,.github,.nyc_output' -s -d build",
```
Comment (benfluleck) [inline package.json]: the dir to build is called dist

**Comment by benfluleck** [comment]
Please implement Babael in the codebase

File: package.json [hunk 2]
```diff
@@ -5,11 +5,18 @@
...
+    "clean": "rm -rf build"
   },
   "author": "Andela Simulations Programme",
   "license": "MIT",
   "dependencies": {
+    "babel-cli": "^6.26.0",
+    "babel-core": "^6.26.3",
+    "babel-plugin-transform-runtime": "^6.23.0",
```
Comment (benfluleck) [inline package.json]: please use babel 7

File: package.json (hunk 1 above)
Comment (mrkingz) [reply package.json]: You are not following the folder structure as agreed. At this point, your index.js (entry point) and babel build should be in the server folder.

**Review by benfluleck — CHANGES_REQUESTED** [review]

File: .babelrc [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+      "@babel/env",
+      {
+        "targets": {
+          "node": "current"
+        }
+      }
+    ]
+  ],
+  "plugins": [
+    "@babel/transform-object-rest-spread",
```
Comment (benfluleck) [inline .babelrc]: Not correct plugin

**Pull request merged**